python/katana/example_data.py

```diff
@@ -54,15 +61,24 @@ def get_input(rel_path) -> Path:
     >>> from katana.local import Graph
     ... graph = Graph(get_input("propertygraphs/ldbc_003"))
     """
-    path = get_inputs_directory() / rel_path
+    path = get_inputs_directory(rel_path=rel_path) / rel_path
     if path.exists():
         return path
     return get_inputs_directory(invalidate=True) / rel_path
 
 
-def get_input_as_url(rel_path) -> str:
+def get_input_as_url(rel_path, datasource="local") -> str:
```

Comment:
It looks like datasource is an enumeration. Make a real enum. https://docs.python.org/3/library/enum.html#creating-an-enum